e2etests/tests/metrics.go

```diff
@@ -223,19 +223,17 @@ var _ = ginkgo.Describe("Metrics", func() {
 		frrs := config.ContainersForVRF(infra.FRRContainers, "")
 		peersConfig := config.PeersForContainers(frrs, ipfamily.IPv4)
 		for i := range peersConfig.PeersV4 {
-			peersConfig.PeersV4[i].Neigh.ToAdvertise.PrefixesWithLocalPref = []frrk8sv1beta1.LocalPrefPrefixes{
```

Comment:
why this was removed?

Reply:
because the webhook denied the resource :)